workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-custom-element/package.json

```diff
@@ -23,11 +23,18 @@
 		"react-router-dom": "^6.15.0",
 		"react-scripts": "5.0.1"
 	},
+	"eslintConfig": {
+		"globals": {
+			"Liferay": true
+		}
+	},
 	"name": "liferay-jethr0-custom-element",
 	"private": true,
 	"scripts": {
 		"build": "react-scripts build",
 		"eject": "react-scripts eject",
+		"format": "liferay-npm-scripts format --experimental-standalone 'src/**'",
+		"lint": "liferay-npm-scripts lint:fix --experimental-standalone 'src/**'",
```

Comment:
@michaelhashimoto I don't think these are necessary. You can run format from the workspaces root `./workspaces` and then run `yarn install` and `yarn format`. It'll SF the entire directory.